docs/manifest.json

```diff
@@ -245,6 +245,12 @@
 		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/packages.md",
 		"parent": null
 	},
+	{
```

Comment:
I think this is meant to be added to `root-manifest.json`. There are local changes on master (removing this) when running `npm run docs:build`.

Reply:
my bad you are right needs to be added to `root-manifest.json`.

Reply:
What's the difference between the two, just so I understand for the future?

Reply:
The `root-manifest.json` file contains all markdown files. The `npm run docs:build` command generates the `manifest.json` file from the `root-manifest.json` file and adds docs from packages, data and components to it.